tokencost/constants.py

```diff
@@ -23,13 +23,19 @@
 # Fetch the latest prices using urllib.request
 PRICES_URL = "https://raw.githubusercontent.com/BerriAI/litellm/main/model_prices_and_context_window.json"
```

Comment:
How did you create this table ?

Reply:
I wrote a utility script with pandas. Never committed it to the repo. Maybe I should!

Reply:
will be amazing if you will post it, we need it for a nightly job to keep everything aligned with the market prices